### PR TITLE
Create Releases “2022-11-08-october-2022”

### PR DESCRIPTION
**Pull request by amanifarooque**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by amanifarooque** [comment]
Preview URL: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/releases/2022-11-08-october-2022/about/updates/releases/october-2022.html

File: _releases/2022-11-08-october-2022.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+last_modified_at: November 8, 2022
+sidenav: about_us
+release-year: "2022"
+description: ""
+---
+## Improvements
+
+* **ElasticSearch Upgrade**: We’ve continued testing and planning for the ElasticSearch upgrade.
+* **Search Engine Results Page (SERP) Redesign Investigations**: We've explored a few options to use more modern frontend technology to power our SERP.
+* **Rails 7 Upgrade**: We've started upgrading components of our search-gov app in preparation for Rails 7.
```
Comment (dawnpm) [inline _releases/2022-11-08-october-2022.md:15]: Please remove the version number. The second mention could become "for the next Rails version/"

**Review by dawnpm — CHANGES_REQUESTED** [review]
just one tiny change, thanks.

File: _releases/2022-11-08-october-2022.md (hunk 1 above)
Comment (amanifarooque) [reply _releases/2022-11-08-october-2022.md]: Updated!

**Review by MyNameIsMissing — DISMISSED** [review]
Looks like a smart apostrophe on line 13.  I guess it isn't really enough to hold up the PR tho.

**Comment by amanifarooque** [comment]
@dawnpm requesting re-review, thank you!

**Review by dawnpm — APPROVED** [review]

**Pull request merged**